src/Adapter/Currency/CommandHandler/BulkDeleteCurrenciesHandler.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
```

Comment:
Don't forget `declare(strict_types=1);` on new files.

Reply:
Big thanks man !
But ... is it really useful in my case  🤔 ?

Reply:
It's a good practice we apply for all new files.

Reply:
Thanks @Progi1984 ... where can I find that in the PS Doc please ?

Reply:
😂 He got you @Progi1984 Seems like you're up for updating the doc 😛

Reply:
Done here: https://github.com/PrestaShop/docs/pull/518

Reply:
Thanks a lot @eternoendless  !